cli/internal/cmd/create.go

```diff
@@ -34,212 +22,24 @@ func NewCreateCmd() *cobra.Command {
 		Short: "Create instances on a cloud platform for your Constellation cluster",
 		Long:  "Create instances on a cloud platform for your Constellation cluster.",
 		Args:  cobra.ExactArgs(0),
-		RunE:  runCreate,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.Flags().Bool("conformance", false, "")
+			cmd.Flags().Bool("skip-helm-wait", false, "")
+			cmd.Flags().Bool("merge-kubeconfig", false, "")
+			cmd.Flags().Duration("timeout", 5*time.Minute, "")
+			// Skip all phases but the infrastructure phase.
+			cmd.Flags().StringSlice("skip-phases", []string{
+				string(skipInitPhase), string(skipAttestationConfigPhase),
+				string(skipCertSANsPhase), string(skipHelmPhase),
+				string(skipImagePhase), string(skipK8sPhase),
+			}, "")
+			return runApply(cmd, args)
```

Comment:
do we plan to deprecate create?

Reply:
Yes, this should be deprecated.
I wasn't sure if we want to do it right away.
We might want to keep it around for a while (even if just as a convenience for devs since `constellation create` is a lost quicker to write than `constellation apply --skip-phases helm,init,attestationConfig,certSANs`)